src/art-styles/circle_sketch.ts

```diff
@@ -0,0 +1,480 @@
+
+    this.paletteIndex = opts.paletteIndex;
+    this.printColors(this.paletteIndex);
+    this.colorPalette = this.generatePalette(this.paletteIndex);
+
+    // this.p5.randomSeed(blockNumber);
+    this.artTypeRandNum = Math.floor(this.p5.random(0, 100)) % 2;
+    this.reseed();
+
+    let bowing = 3;
+    let roughness = 4;
+    let maxOffset = 3;
+    switch (this.gridSize) {
+      case 3:
+        bowing = 2;
+        roughness = 3;
+        maxOffset = 2;
+        this.fillGap = 6;
+        this.strokeWidthFill = 5;
+        this.strokeWidth = 3;
+        this.strokeWidthOutline = 3;
+        break;
+      case 6:
+        bowing = 3;
+        roughness = 2;
+        maxOffset = 1;
+        this.fillGap = 5;
+        this.strokeWidthFill = 3.5;
+        this.strokeWidth = 2;
+        this.strokeWidthOutline = 2;
+        break;
+      case 9:
+        bowing = 3;
+        roughness = 2;
+        maxOffset = 1;
+        this.fillGap = 4;
+        this.strokeWidthFill = 2.5;
+        this.strokeWidth = 1;
+        this.strokeWidthOutline = 1;
+        break;
+      case 12:
+        bowing = 2;
+        roughness = 2;
+        maxOffset = 1;
+        this.fillGap = 3.5;
+        this.strokeWidthFill = 2;
+        this.strokeWidth = 0.5;
+        this.strokeWidthOutline = 0.5;
+        break;
+      default:
+        bowing = 3;
+        roughness = 3;
+        maxOffset = 1;
+        this.fillGap = 5;
+        this.strokeWidthFill = 3;
+        this.strokeWidth = 0.5;
+        this.strokeWidthOutline = 0.5;
+        break;
+    }
+
+    this.sizeOfBox = (squareSize - this.strokeWidthOutline) / this.gridSize;
+
+    if (this.noFill) {
+      this.strokeColor = this.p5.color(
+        this.colorTable.getNum(this.paletteIndex, 0),
+        this.colorTable.getNum(this.paletteIndex, 1),
+        this.colorTable.getNum(this.paletteIndex, 2)
+      );
+      switch (this.gridSize) {
+        case 3:
+          this.strokeWidth = 12;
+          break;
+        case 6:
+          this.strokeWidth = 8;
+          break;
+        case 9:
+          this.strokeWidth = 6;
+          break;
+        case 12:
+          this.strokeWidth = 4;
+          break;
+        default:
+          this.strokeWidth = 1.5;
+          break;
+      }
+    } else {
+      this.strokeColor = this.p5.color(10);
+    }
+
+    this.iterator = 1;
+    this.lineIterator = 0;
+    this.firstLine = true;
+    this.triangleIterator = 0;
+    this.allTriangles = [];
+    this.scribble = new Scribble(this.p5, bowing, roughness, maxOffset);
+    this.fillLines = [];
+    this.fillIterator = 0;
+    this.fillDone = true;
+    this.triangleFillColors = [];
+    this.paper = null;
+
+    this.paperIndex = opts.paperIndex;
+  }
+
+  setup(canvasParentRef: Element) {
+    this.p5
+      .createCanvas(this.canvasWidth, this.canvasHeight)
+      .parent(canvasParentRef);
+
+    this.p5.frameRate(120); // highest possible framerate
+    this.p5.loadImage(
```

Comment:
why are u even loading image?

Reply:
Its not needed actually, will push cleanup commits soon